addons/stock/report/stock_lot_customer.py

```diff
@@ -48,6 +48,12 @@ def _from(self):
             ON state.id = partner.state_id
             LEFT JOIN res_country AS country
             ON country.id = partner.country_id
+            LEFT JOIN stock_picking AS picking_return
+            ON picking_return.return_id = sml.picking_id
+            LEFT JOIN stock_move_line AS sml_return
+            ON sml_return.picking_id = picking_return.id 
+            AND sml_return.lot_id = lot.id 
+            AND sml_return.state = 'done'
```

Comment:
```suggestion
                AND sml_return.lot_id = lot.id 
                AND sml_return.state = 'done'
```
Si it's more readable :)